lib/jsonpath/enumerable.rb

```diff
@@ -27,10 +20,6 @@ def each(context = @object, key = nil, pos = 0, &blk)
         each(context, key, pos + 1, &blk) if node == @object
       when /^\[(.*)\]$/
         handle_wildecard(node, expr, context, key, pos, &blk)
-      else
```

Comment:
I literary found no use for this code segment. 

Let it be said here, if anybody has any problems with this missing segment, submit an issue, and I'll investigate use cases for that part.